src/include/block_chunkcache.h

```diff
@@ -68,5 +68,6 @@ struct __wt_chunkcache {
     char *dev_path;             /* the storage path if we are on a file system or a block device */
     unsigned int evict_trigger; /* When this percent of cache is full, we trigger eviction. */
     unsigned int hashtable_size;
-    int type; /* location of the chunk cache (volatile memory or file) */
+    int type;              /* location of the chunk cache (volatile memory or file) */
+    char **pinned_objects; /* list of object we wish to pin in chunk cache */
```

Comment:
Nitpick: "object" -> "objects"

Reply:
Update in [5e9ac89](https://github.com/wiredtiger/wiredtiger/pull/9415/commits/5e9ac89d77983d0f50b96a1d984098d292cc517a)